python/python/ert_gui/simulation/detailed_progress.py

```diff
@@ -257,27 +266,23 @@ def reload(self, file_name):
         self.show()
 
 
-class DetailedProgressDialog(QDialog):
+class DetailedProgressDialog(QWidget):
```

Comment:
Well, we could also consider changing the class name ...

Reply:
Yes we should :)